### PR TITLE
(task:7102) add script which translate normative template into kubernetes YAML

### PR DESCRIPTION
**Pull request by princessfruitt**

### CODE REVIEW

File: TOSCA_kubernetes_defenition_1_0.YAML [hunk 1]
```diff
@@ -1,4 +1,4 @@
-tosca_definitions_version: tosca_simple_yaml_1_0
+tosca_definitions_version: tosca_simple_yaml_1_2
```
Comment (bura2017) [inline TOSCA_kubernetes_defenition_1_0.YAML]: ?

File: k8stranslator/k8sCreate.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                      prop.name != API_VERSION and prop.name != API_GROUP]
+        k8s_list.append(
+            KubernetesObj(template.entity_tpl.get(TYPE).split('.'),
+                          template.get_property_value(API_GROUP) + '/' + template.get_property_value(API_VERSION),
+                          props_list))
+    for k8s_obj in k8s_list:
+        complete_yaml = yaml.dump([dict(
+            apiVersion=k8s_obj.get_version(),
+            kind=k8s_obj.get_kind()[2],
+            spec=k8s_obj.get_props()
```
Comment (bura2017) [inline k8stranslator/k8sCreate.py]: Strongly depends on TOSCA templates, as if adding new property requires changing the code

File: k8stranslator/k8sCreate.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    k8s_list = []
+    for template in node_templates:
+        props_list = [prop.value for prop in template.get_properties_objects() if
+                      prop.name != API_VERSION and prop.name != API_GROUP]
+        k8s_list.append(
+            KubernetesObj(template.entity_tpl.get(TYPE).split('.'),
+                          template.get_property_value(API_GROUP) + '/' + template.get_property_value(API_VERSION),
+                          props_list))
+    for k8s_obj in k8s_list:
+        complete_yaml = yaml.dump([dict(
```
Comment (bura2017) [inline k8stranslator/k8sCreate.py]: does it work for multiple objects?

File: TOSCA_kubernetes_defenition_1_0.YAML [hunk 4]
```diff
@@ -8,22 +8,42 @@ metadata:
 description: Tosca definition for kubernetes
 
 node_types:
+
   kubernetes.nodes.Root:
     derived_from: tosca.nodes.Root
+    properties:
```
Comment (bura2017) [inline TOSCA_kubernetes_defenition_1_0.YAML]: Also I offer to add metadata object here as it's used for object names. Look this https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields

File: k8stranslator/k8sCreate.py (hunk 3 above)
Comment (princessfruitt) [reply k8stranslator/k8sCreate.py]: +

File: TOSCA_kubernetes_defenition_1_0.YAML (hunk 4 above)
Comment (princessfruitt) [reply TOSCA_kubernetes_defenition_1_0.YAML]: +

File: TOSCA_kubernetes_defenition_1_0.YAML (hunk 1 above)
Comment (princessfruitt) [reply TOSCA_kubernetes_defenition_1_0.YAML]: Opps

File: k8stranslator/k8sCreate.py (hunk 2 above)
Comment (princessfruitt) [reply k8stranslator/k8sCreate.py]: +

**Pull request merged**